libavfilter/dnn_interface.h

```diff
@@ -30,6 +30,8 @@
 #include "libavutil/frame.h"
 #include "avfilter.h"
 
+#define DNN_GENERIC_ERROR FFERRTAG('D','N','N','!')
+
 typedef enum {DNN_SUCCESS, DNN_ERROR} DNNReturnType;
```

Comment:
how about to remove DNNReturnType?

returns 0 for success, and negative values for errors.

Reply:
Sure. That's seems better since `DNN_ERROR` isn't used anywhere after these changes.